src/codecs/webp/encoder.ts

```diff
@@ -1,13 +1,76 @@
 import EncoderWorker from './Encoder.worker';
 
-export interface EncodeOptions { quality: number; }
+export enum WebPImageHint {
+  WEBP_HINT_DEFAULT, // default preset.
+  WEBP_HINT_PICTURE, // digital picture, like portrait, inner shot
+  WEBP_HINT_PHOTO,   // outdoor photograph, with natural lighting
+  WEBP_HINT_GRAPH,   // Discrete tone image (graph, map-tile etc).
+}
+
+export interface EncodeOptions {
```

Comment:
If at all possible, we should link to a resource somewhere explaining what these options are (maybe even from the UI?)

Reply:
I'll add a comment saying which files have details. There's the header which has some info, and the help output which has slightly different info.

Related: we should find a way to add info into the UI. I guess it's a design job.

Reply:
Doh, you said UI too. Agreed. I think this is important. Want me to add details for this PR or create an issue.

I'll detail which files the headers are in as part of this PR.

Reply:
Filed https://github.com/GoogleChromeLabs/squoosh/issues/111